commands/alias.go

```diff
@@ -101,7 +101,9 @@ func alias(command *Command, args *Args) {
 		var eval string
 		switch shell {
 		case "fish":
-			eval = `eval (hub alias -s)`
+			eval = `function git --description 'Alias for Github wrapper for git called hub, that adds some Github sugar to git.'
```

Comment:
You seem to have only updated the description text in the test but not here as well. Now they don't match and the tests fail.